test/end-to-end/cases-coverage/xsl-choose-01.xsl

```diff
@@ -14,15 +14,15 @@
         </xsl:when>
         <xsl:otherwise version="3.0">                                          <!-- Expected miss -->
           <node type="choose otherwise" version="3.0">                         <!-- Expected miss -->
-            <xsl:text version="3.0">200</xsl:text>                             <!-- Expected miss -->
+            <xsl:text version="3.0">0</xsl:text>                               <!-- Expected miss -->
```

Comment:
I changed some of the numbers to avoid confusion, such as whether a 200 in the result came from here or line 30.